search.py

```diff
@@ -49,6 +50,44 @@ def dfs(graph):
     # If there is no path from source cell to goal cell than return the string below
     return "Failure: No Path"
 
+
+def bfs(graph):
+    #creates a queue the size of the board so we have enough space
+    dim = len(graph)
+    queue = Queue(maxsize=(dim**2))
+
+    # Create a visited array of same dimensions as the graph to ensure that BFS
```

Comment:
Fix this comment, as BFS won't run forever even without a visited array but this will ensure that fringe size will be minimal instead.